src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -540,7 +540,7 @@ case class JesBackend(actorSystem: ActorSystem)
 
       status match {
         case Run.Success(events) if backendCall.runtimeAttributes.failOnStderr && stderrLength.intValue > 0 =>
-          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength")).future
+          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption).future
```

Comment:
Ok - this won't change the existing behavior as `FailedExecutionHandle` has a default value `None` on returnCode anyway.